site/topics/selection/selection.rst

```diff
@@ -109,6 +111,124 @@ Tournament Selection
 Fitness Proportional Selection
 ==============================
 
+* There are a collection of selection strategies that are fitness proportaional
+
+    * The probability of selecting an indiviual depends on its fitness value compared to the whole populations's fitness
+
+
+* The simplest of these is to have the probability be the individual's proportion of the total population fitness
+
+    * :math:`p(i) = \frac{f(i)}{\sum^{\mu}_{j=1}f(j)}`
+    * Where :math:`p(i)` is the probabability of selecting individual :math:`i`
+    * And :math:`f(i)` is the fitness of individual :math:`i`
+    * Note that the sum of the probabilities must be one :math:`\sum^{\mu}_{j=1}p(j) = 1`
+
+
+* The benefit of such a selection strategy is that it's very *fair*, which is valuable
```

Comment:
I'm a little wary of using the word _fair_ here. With all the analogies to populations, suggesting that more fit individuals are more likely to be selected may stoke some political beliefs unnecessarily.

Staying objective, and avoiding shorthand explanations may be more sensible. For example, saying something like: selecting higher fitness solutions may help produce even better solutions; but may also propell to a local maximum; selecting lower fitness can help with keeping the solution population diverse